pkg/ingester/ingester_v2.go

```diff
@@ -355,6 +358,9 @@ type TSDBState struct {
 	forceCompactTrigger chan chan<- struct{}
 	shipTrigger         chan chan<- struct{}
 
+	// Timeout chosen for idle compactions.
+	compactionIdleTimeout time.Duration
```

Comment:
Should this be per-`userTSDB`? I figured it might be overkill, but open to opinions.

Reply:
A global should be fine.